### PR TITLE
Add paper on code and API mining.

### PR DESCRIPTION
**Pull request by timmolderez**
Signed-off-by: Tim Molderez <id@timmolderez.be>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +188,7 @@ Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces
 
 * <img src="badges/4-pages-gray.svg" alt="4-pages" align="top"> [Hierarchical Learning of Cross-Language Mappings through Distributed Vector Representations for Code](https://bdqnghi.github.io/files/icse-18-nier.pdf) - Nghi D. Q. Bui, Lingxiao Jiang, ICSE 2018.
 * <img src="badges/7-pages-gray.svg" alt="7-pages" align="top"> [DeepAM: Migrate APIs with Multi-modal Sequence to Sequence Learning](https://arxiv.org/abs/1704.07734v1) - Xiaodong Gu, Hongyu Zhang, Dongmei Zhang, Sunghun Kim, IJCAI 2017.
+* <img src="badges/9-pages-gray.svg" alt="9-pages" align="top"> [Mining Change Histories for Unknown Systematic Edits](https://ieeexplore.ieee.org/abstract/document/7962375) - Tim Molderez, Reinout Stevens, Coen De Roover, MSR 2017.
```
Comment (m09) [inline README.md]:
```suggestion
* <img src="badges/9-pages-gray.svg" alt="9-pages" align="top"> [Mining Change Histories for Unknown Systematic Edits](http://soft.vub.ac.be/Publications/2017/vub-soft-tr-17-04.pdf) - Tim Molderez, Reinout Stevens, Coen De Roover, MSR 2017.
```

**Review by m09 — CHANGES_REQUESTED** [review]
Hi @timmolderez, thanks for the PR! I suggested an alternative link since IEEE is paywalled.

File: README.md (hunk 1 above)
Comment (timmolderez) [reply README.md]: Ah yes, you're right; that link is much better :) Thanks!

**Comment by m09** [comment]
Oh and sorry I'm only now realizing that you are first author. We organize [reading clubs](https://github.com/src-d/reading-club) regularly and your paper is extremely relevant to what we are doing at source{d}, let me know if you'd like to present your paper there, it would be really cool :)

**Review by m09 — APPROVED** [review]

**Comment by timmolderez** [comment]
I'd be happy to present the paper at the reading club! Perhaps I should mention though that the work doesn't really use machine learning directly; it mainly applies data mining techniques to find patterns in a software project's history.

**Comment by vmarkovtsev** [comment]
@timmolderez This is very cool work, now I want to add it to our [Hercules](https://github.com/src-d/hercules) badly.

**Comment by m09** [comment]
@timmolderez very cool :tada: Would our next session (Friday, November 30, 4pm CET) be fine for you?

**Comment by timmolderez** [comment]
Thanks @m09 @vmarkovtsev :) Yes, Nov 30th at 4pm CET is perfect! The session will take place in a Zoom meeting?

**Comment by m09** [comment]
@timmolderez Perfect! Yes, it's a zoom meeting (ID 974-346-848) :)

**Comment by timmolderez** [comment]
Great! See you next week Friday :)

**Pull request merged**